src/frame.js

```diff
@@ -0,0 +1,38 @@
+var Frame = function(){
+};
+
+Frame.prototype.roll1 = function(score) {
+  this.roll1Score = score;
+  return this.roll1Score;
+};
+
+Frame.prototype.roll2 = function(score) {
+  this.roll2Score = score;
+  return this.roll2Score
+};
+
+Frame.prototype.frameScore = function() {
+  var totalScore = (this.roll1Score + this.roll2Score);
+  return totalScore;
+};
+
+Frame.prototype.strike = function() {
+  if(this.roll1Score === 10){
+    return true
+  }
+  else{
+    return false
+  }
+};
+
+Frame.prototype.spare = function() {
+  if(this.roll1Score + this.roll2Score === 10){
+    return true
+  }else{
+    return false
+  }
+};
+
```

Comment:
These two functions don't need to be an if/else statement - they can be rewritten as
```javascript
Frame.prototype.strike = function() {
 return this.roll1Score === 10
}
Frame.prototype.spare = function() {
  return this.roll1Score + this.roll2Score === 10
}
```